TonicCertificateGenerator/Program.cs

```diff
@@ -25,8 +25,8 @@ private static void Main(string[] args)
                 contactManager.SetOutputDirectoryForFiles(config["OUTPUT_DIR"]);
                 var listContactFilepathPair = contactManager.GetDocumentForAllContacts();
                 var mailManager = (IMailManager) serviceProvider.GetService(typeof(IMailManager));
-                foreach (var contactFilepathPair in listContactFilepathPair)
-                    mailManager.SendEmailToContactWithAttachmnent(contactFilepathPair);
+                //foreach (var contactFilepathPair in listContactFilepathPair)
```

Comment:
why comments?

Reply:
Comments is removed.
It was a small mistake.